cognite/neat/core/extractors/graph_sheet_to_graph.py

```diff
@@ -0,0 +1,132 @@
+import logging
+import warnings
+
+import pandas as pd
+from rdflib import RDF, Graph, Literal
+
+from cognite.neat.core.data_classes.transformation_rules import TransformationRules
+
+
```

Comment:
Just a general comment on the structure of this file. There are three functions `validate_if_graph_capturing_sheet_empty`, `validate_rules_graph_pair` and `sheet2graph`. This is a soap box for me, but I suggest reversing the order in which these functions appear. 

I think the most important function should appear at the top of the file, `sheet2graph`, and then the two helper functions after that. This is because this is the natural order to read it.